IFC4x3/Sections/Domain specific data schemas/Schemas/IfcTunnelDomain/Entities/IfcTunnel/DocEntity.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcTunnel" UniqueId="56b83b24-edac-4879-b81d-3144d075358e" BaseDefinition="IfcFacility" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="2da2e39c-0999-4bfc-b5a3-e7006c64b7b8" DefinedType="IfcTunnelTypeEnum" AttributeFlags="1">
```

Comment:
Should be mandatory and rule changed (the NOT EXISTS part is then not necessary).

Reply:
Should be fixed now.